unified-codes/apps/web/src/components/templates/Explorer.tsx

```diff
@@ -23,9 +29,31 @@ export interface ExplorerProps {
   onUpdateVariables: (variables: IExplorerVariables) => void;
 }
 
+const getStyles = (theme: ITheme) => {
+  const borderStyle = `1px solid ${theme.palette.divider}`;
+  return {
+    pagination: { backgroundColor: theme.palette.background.toolbar },
+    root: { backgroundColor: theme.palette.background.default, maxHeight: '100%', maxWidth: 900 },
+    searchBar: { paddingLeft: 15 },
+    table: {
+      marginTop: 5,
+      maxHeight: `calc(100vh - ${FOOTER_HEADER_HEIGHT}px)`,
+      overflowY: 'scroll' as overflow,
+      '& th': { backgroundColor: theme.palette.background.toolbar, fontWeight: 700 },
+      '& thead > tr': { borderBottom: borderStyle },
+      '& tr > th': { borderRight: borderStyle },
+      '& tr > td': { borderRight: borderStyle, borderBottom: 0 },
+      '& tr > td:last-child': { borderRight: 0 },
+      '& tr > th:last-child': { borderRight: 0 },
+      '& tr > td:first-child': { fontWeight: 700 },
```

Comment:
Thinking some of this could potentially be abstracted to a shared styles file/folder? Would you be thinking to refactor away these hardcoded style constants and stuff?